pkg/networkpolicy/controller.go

```diff
@@ -665,6 +665,13 @@ func (c *Controller) syncNFTablesRules(ctx context.Context) error {
 				"ct", "state", "established,related", "accept"),
 		})
 
+		// only process protocols supported by Kubernetes network policies
+		tx.Add(&knftables.Rule{
+			Chain: chainName,
+			Rule: knftables.Concat(
+				"meta", "l4proto", "!=", "{ tcp, udp, sctp }", "accept"),
```

Comment:
ping @danwinship

Reply:
(There's no real reason to use `knftables.Concat` here. You can just write it as a single string.)